core/src/main/java/org/keycloak/sdjwt/SdJwt.java

```diff
@@ -198,14 +199,18 @@ public List<String> getDisclosures() {
     /**
      * Verifies SD-JWT as to whether the Issuer-signed JWT's signature and disclosures are valid.
      *
-     * @param verificationOpts Options to parameterize the Issuer-Signed JWT verification. A verifier
-     *                         must be specified for validating the Issuer-signed JWT. The caller
-     *                         is responsible for establishing trust in that associated public keys
-     *                         belong to the intended issuer.
+     * @param issuerVerifyingKeys Verifying keys for validating the Issuer-signed JWT. The caller
+     *                            is responsible for establishing trust in that the keys belong
+     *                            to the intended issuer.
+     * @param verificationOpts    Options to parameterize the Issuer-Signed JWT verification.
      * @throws VerificationException if verification failed
      */
-    public void verify(IssuerSignedJwtVerificationOpts verificationOpts) throws VerificationException {
-        new SdJwtVerificationContext(issuerSignedJWT, disclosures).verifyIssuance(verificationOpts);
+    public void verify(
+            List<SignatureVerifierContext> issuerVerifyingKeys,
+            IssuerSignedJwtVerificationOpts verificationOpts
+    ) throws VerificationException {
+        new SdJwtVerificationContext(issuerSignedJWT, disclosures)
```

Comment:
Why is the context inline created? Since disclosures and issuerSignedJWT final, wouldn't  it make more sense to create the context also directly in the constructor?